src/main/resources/application.yml

```diff
@@ -32,8 +32,8 @@ spring:
     max-in-memory-size: 24MB
   data:
     mongodb:
-      uri: ${MONGO_URI:mongodb://${DBUSER:root}:${DBPASS:E5press0}@localhost:27017/terminology}?authSource=admin&maxPoolSize=50&connectTimeoutMS=2000&serverSelectionTimeoutMS=2000
+      uri: mongodb://root:E5press0@${MONGO_HOST:localhost}:27017/madie?authSource=admin&maxPoolSize=50&connectTimeoutMS=2000&serverSelectionTimeoutMS=2000
```

Comment:
Are we changing database as well?

Reply:
Nice catch, I committed too many changes.